src/gui/advancedsettings.cpp

```diff
@@ -82,6 +82,7 @@ enum AdvSettingsRows
     DISK_CACHE_TTL,
     OS_CACHE,
     GUIDED_READ_CACHE,
+    COALESCE_RW,
```

Comment:
But this doesn't take effect in non-Windows builds. Nor in builds using libtorrent < 1.0.7

Reply:
>But this doesn't take effect in non-Windows builds. Nor in builds using libtorrent < 1.0.7

Sorry. I read the code wrong. But it doesn't take into account builds against libtorrent < 1.0.7.
Same goes for all other instances.

Reply:
You must have meant libtorrent 1.1.7.

You must have noticed there is a guard (which hides the option) at: https://github.com/qbittorrent/qBittorrent/pull/8336/files#diff-634bac168faf10189e2a482c55a251c3R321

Isn't it enough? or I should change it to runtime check with `libtorrent::version()`?

And also I don't think it is much of a problem to always [apply settings](https://github.com/qbittorrent/qBittorrent/pull/8336/files#diff-5961e6811ef7cd17f473ca94c53d5559R1312 ), as it is doing nothing in libtorrent < 1.1.7.